docker/compose/docker-compose_hadoop284_hive233_spark244.yml

```diff
@@ -231,7 +259,7 @@ services:
       - 8126:8126
 
   adhoc-1:
-    image: apachehudi/hudi-hadoop_2.8.4-hive_2.3.3-sparkadhoc_2.4.4:latest
+    image: yihua/hudi-hadoop_2.8.4-hive_2.3.3-sparkadhoc_2.4.4:latest
```

Comment:
Is `yihua` docker account only meant for testing? Shall we keep `apachehudi`?

Reply:
Yes, I got the access to apachehudi account and updated the image ref.